docs/fundamentals/flow/monitoring-flow.md

```diff
@@ -156,40 +156,50 @@ Because not all Pods have the same role, they expose different kinds of metrics:
+| `jina_return_client_request_bytes`  | [Summary](https://prometheus.io/docs/concepts/metric_types/#summary)   | Measures the size of the request in Bytes return from the Gateway to the Client                                     |
 
 ```{seealso} 
 You can find more information on the different type of metrics in Prometheus [here](https://prometheus.io/docs/concepts/metric_types/#metric-types)
 ```
 
 ### Head Pods
 
-| Metrics name                       | Metrics type                                                          | Description                                                                                                     |
-|------------------------------------|-----------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
-| `jina_receiving_request_seconds`   | [Summary](https://prometheus.io/docs/concepts/metric_types/#summary)  | Measure the time elapsed between receiving a request from the gateway and sending back the response.        |
-| `jina_sending_request_seconds`     | [Summary](https://prometheus.io/docs/concepts/metric_types/#summary)  | Measure the time elapsed between sending a downstream request to an Executor and receiving the response back.   |
+| Metrics name                            | Metrics type                                                            | Description                                                                                                   |
+|-----------------------------------------|-------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
+| `jina_receiving_request_seconds`        | [Summary](https://prometheus.io/docs/concepts/metric_types/#summary)    | Measure the time elapsed between receiving a request from the gateway and sending back the response.          |
+| `jina_sending_request_seconds`          | [Summary](https://prometheus.io/docs/concepts/metric_types/#summary)    | Measure the time elapsed between sending a downstream request to an Executor and receiving the response back. |
+| `jina_sending_request_bytes`            | [Summary](https://prometheus.io/docs/concepts/metric_types/#summary)    | Measure the size of the downstream requests send to an Executor in Bytes                                      |
+| `jina_failed_requests_total`            | [Counter](https://prometheus.io/docs/concepts/metric_types/#counter)    | Counts the number of failed requests returned by the gateway                                                  |
+| `jina_send_request_bytes`               | [Summary](https://prometheus.io/docs/concepts/metric_types/#summary)    | Measures the size in Bytes of the request send by the Head to the Pod                                         |
```

Comment:
`sent` is the right word and not `send`.

`jina_send_request_bytes` is from the `Head` to the `Executor`?

Reply:
I change the documentation so that it is clear